build-scripts/puddle-conf/atomic_openshift-3.0.conf

```diff
@@ -13,4 +13,4 @@ emails = aos-devel@redhat.com,aos-qe@redhat.com
 announcer = openshift-puddle <aos-devel@redhat.com>
 publish = no
 keys = fd431d51,f21541eb
-blacklist = no
\ No newline at end of file
+denylist = no
```

Comment:
in puddle confs these are keywords.

of course, hopefully we won't need puddles much longer...